action.go

```diff
@@ -295,21 +295,27 @@ func (c *cmdAction) ClusterCanRetry() bool {
 // set Nil to true. If not the return value will be unmarshaled into Rcv
 // normally.
 type MaybeNil struct {
-	Nil bool
-	Rcv interface{}
+	Nil   bool
+	Empty bool
```

Comment:
I'd like to name the field `EmptyArray`, to make it a bit more specific

Reply:
I also thought about check for empty string here but in tests reading of `$0\r\n` causes `rm.UnmarshalRESP` to return EOF error.
If there is no ways to check for empty string I'll of course rename property to `EmptyArray`

Reply:
That's weird about the tests. In any case, I'd rather stick with just `EmptyArray` for now, as it's the minimum that we're sure we need at the moment. `EmptyBulkString`, `EmptySimpleString`, etc... can be added alongside it at a later date when it's a need that comes up.